src/utils/border.scss

```diff
@@ -7,31 +7,49 @@ $borders: (
   'Top': top
 );
 
+$border-style: 1px solid $cu-divider--light;
```

Comment:
since we have a standard and slightly lighter border, think we need differentiate between the two here?  or will this be specific to use within tab panels/tables?